scripts/upload.sh

```diff
@@ -10,6 +12,7 @@ if [[ -f ${key_file} ]]; then
 fi
 trap 'rm -f ${key_file}; ssh-agent -k' EXIT
 
+mkdir -p ~/.ssh
```

Comment:
Useful to understand what happens

```suggestion
mkdir -vp ~/.ssh
```